config/initializers/cors.rb

```diff
@@ -0,0 +1,16 @@
+# Be sure to restart your server when you modify this file.
+
+# Avoid CORS issues when API is called from the frontend app.
+# Handle Cross-Origin Resource Sharing (CORS) in order to accept cross-origin Ajax requests.
+
+# Read more: https://github.com/cyu/rack-cors
+
+# Rails.application.config.middleware.insert_before 0, Rack::Cors do
+#   allow do
+#     origins "example.com"
+#
+#     resource "*",
+#       headers: :any,
+#       methods: [:get, :post, :put, :patch, :delete, :options, :head]
+#   end
+# end
```

Comment:
- Please 🙏🏽 uncomment these lines to avoid CORS conflicts and be able to see the greetings in the Front end.

![image](https://github.com/hafiz1379/hello-rails-back-end/assets/58642949/6e9a86d1-a306-4658-b4a6-72af75aec8e3)


```ruby
 Rails.application.config.middleware.insert_before 0, Rack::Cors do
   allow do
     origins "*"

     resource "*",
       headers: :any,
       methods: [:get, :post, :put, :patch, :delete, :options, :head]
   end
 end
```